### PR TITLE
It2 season helper methods shots

### PR DESCRIPTION
**Pull request by hopesgit**
Added:

- total_shots_per_team method
- shots_to_goals_ratio_per_team method
- related tests

### CODE REVIEW

File: lib/stat_tracker.rb
Comment (AngelaGuardia) [inline lib/stat_tracker.rb]: Really cool use of merge! I had not seen that before. I think we can name the block variables something more descriptive `|team_id, shots, goals|`

Also within the division you only need to make one value a float then the whole think will evaluate to a float.

File: lib/stat_tracker.rb
Comment (AngelaGuardia) [inline lib/stat_tracker.rb:105]: Your merge caused one `end` to disappear here!

**Review by AngelaGuardia — CHANGES_REQUESTED** [review]
Really well done Hope. Left some improvement suggestions for you!

File: lib/stat_tracker.rb
Comment (hopesgit) [reply lib/stat_tracker.rb]: Have changed it so that only one is a float now!

File: lib/stat_tracker.rb
Comment (hopesgit) [reply lib/stat_tracker.rb:105]: Thank you, I've fixed this too

File: lib/stat_tracker.rb
Comment (leahriffell) [reply lib/stat_tracker.rb]: Oh interesting! I've never seen this before either. Cool!

**Review by leahriffell — APPROVED** [review]
Looks good to me, Hope! Once the other PR is merged (rspec bug), we'll just have to update your assertions so that team_id is a string instead of an integer. We could merge that other PR first if it makes it easier to just update in this PR instead of having to do a separate branch.

File: lib/stat_tracker.rb
Comment (sarynm12) [reply lib/stat_tracker.rb]: That is interesting! Never used merge before, either, but very cool to see!

**Review by sarynm12 — APPROVED** [review]

File: lib/stat_tracker.rb
Comment (hopesgit) [reply lib/stat_tracker.rb]: Yup, I remembered seeing a "merge" command during my many scans of ruby-docs and I thought it would be cool to use since I was working with two different hashes. Turns out, it's good!

**Review by AngelaGuardia — APPROVED** [review]
Thanks for making those changes @hopesgit . I will hold off on merging this as per @leahriffell's suggestion.

**Comment by leahriffell** [comment]
Thanks @guardiac! In chatting with Hope, it may make more sense to merge this first and then I can fix the string issues in my PR to avoid Hope having a million merge conflicts.

**Pull request merged**
